pyvista/plotting/helpers.py

```diff
@@ -86,7 +95,8 @@ def plot(var_item, off_screen=None, full_screen=False, screenshot=None,
     Returns
     -------
     cpos : list
-        List of camera position, focal point, and view up.
+        List of camera position, focal point, and view up.  Returned
+        only when ``return_cpos=True``.
 
     img : numpy.ndarray
         Array containing pixel RGB and optionally alpha values.
```

Comment:
I think we need to add a third optional `viewer` for `return_viewrr=True`. This is important to know the order of multiple return values.

And this was done earlier, but on line 110 "Returned only when \`\`screenshot=True\`\`" is probably not the whole truth: we also need to consider `return_img`...

Reply:
32f214c51f92965a3ed706dfd8c5ee2d58c9db8e

Reply:
See the comment for `show()` about clarifying the return signature.